build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 
 apply plugin: 'java'
 
-def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "8.7.9"
+def buildNumber = System.getenv("RELEASE_VERSION")?.replace("refs/tags/", "") ?: "8.7.10-prerelease-HMAN-493"
```

Comment:
This should be updated to match final version number before being merged to `master`.

Reply:
Solved in [5defa97](https://github.com/hmcts/befta-fw/pull/195/commits/5defa970a0b990aa305345c501110e42211ef429).